src/world.cpp

```diff
@@ -31,6 +35,139 @@ void World::init_materials(Json::Value material_data){
     }
 }
 
+void World::Get_Mesh_Greedy(std::vector<unsigned short> &indices,
```

Comment:
METHODS ARE SNAKE CASE

Also this might be better off as a friend function in its own file, rather than a method.